codespell_lib/data/dictionary.txt

```diff
@@ -2653,6 +2653,8 @@ athron->athlon
 athros->atheros
 atleast->at least
 atll->all
+atmoic->atomic
```

Comment:
Maybe also `atmo->atom` and its plural? :shrug:

Reply:
I'm not sure about this one. There exist vlc, mozilla and Bosch plugins/products with this name.

Maybe Peter has to help us sorting here.

Reply:
Fair enough, feel free to ignore :+1:

Reply:
Yeah people seem to use it to shorten atmosphere too. A quick random look at grep.app only found valid usage.